SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -553,73 +634,46 @@ Plot {
 			}
 		}
 	}
-
-	prGetTickLabelSize { |drawGrid|
-		var params, charCnt, labelSize;
-		var gridEdges = if ( drawGrid.isKindOf(DrawGridY) ) {
-			[drawGrid.bounds.top, drawGrid.bounds.bottom]
-		} {
-			[drawGrid.bounds.left, drawGrid.bounds.right]
-		};
-
-		params = drawGrid.grid.getParams(
-			drawGrid.range[0], drawGrid.range[1], gridEdges[0], gridEdges[1]
-		);
-		charCnt = try {
-			params.labels.flatten.collect(_.size).maxItem;
-		} { 3 };
-		labelSize = try {
-			("0" ! charCnt).join.bounds(font).size
-		} {
-			("0" ! charCnt).join.bounds(Font(Font.defaultSansFace, 9)).size
-		};
-		labelSize.width = labelSize.width * 1.1;
-
-		^[labelSize, params.labels]
-	}
 }
 
 
-
 Plotter {
 
-	var <>name, <>bounds, <>parent;
+	var <>name;
+	var <>bounds, <>parent;
 	var <value, <data, <domain;
-	var <plots, <specs, <domainSpecs, plotColor, labelX, labelY;
+	var <plots, <specs, <domainSpecs, plotColor;
 	var <cursorPos, plotMode, <>editMode = false, <>normalized = false;
 	var <>resolution = 1, <>findSpecs = true, <superpose = false;
 	var modes, <interactionView;
 	var <editPlotIndex, <editPos;
-
 	var <>drawFunc, <>editFunc;
+	var axisLabelX, axisLabelY;
 
 	*new { |name, bounds, parent|
 		^super.newCopyArgs(name).makeWindow(parent, bounds)
 	}
 
 	makeWindow { |argParent, argBounds|
-		var btnBounds;
 		parent = argParent ? parent;
 		bounds = argBounds ? bounds;
+
 		if(parent.isNil) {
 			parent = Window.new(name ? "Plot", bounds ? Rect(100, 200, 400, 300));
-			bounds = parent.view.bounds.insetBy(8);
+			bounds = parent.view.bounds.insetBy(5);
 			parent.background_(QtGUI.palette.window);
 			interactionView = UserView.new(parent, bounds);
 
 			interactionView.drawFunc = { this.draw };
 			parent.front;
 			parent.onClose = { parent = nil };
-
 		} {
 			bounds = bounds ?? { parent.bounds.moveTo(0, 0) };
 			interactionView = UserView.new(parent, bounds);
 			interactionView.drawFunc = { this.draw };
 		};
-		modes = [\points, \levels, \linear, \plines, \steps, \bars].iter.loop;
-		this.plotMode = \linear;
-		this.plotColor = GUI.skins.plot.plotColor;
 
+		// set up interactionView
```

Comment:
this comment is redundant, I think?

Reply:
BTW, it makes sense to me for this kind of "helper" method to be private, but if you have thoughts on that, just let me know. Also whether these private methods should live close to the method they're helping, or rather gathered at the bottom of the class definition.